tensorrt_llm/serve/encode_batcher.py

```
@@ -0,0 +1,181 @@
+        token_ids_batch = [request.token_ids for request in batch]
+        try:
+            # encode_fn is synchronous and blocking (a GPU forward); run it in the
+            # default executor so the event loop keeps serving requests.
+            results = await loop.run_in_executor(None, self._encode_fn, token_ids_batch)
+        except Exception as exc:  # noqa: BLE001 - fail only this batch
+            # Isolate the failure to this batch; keep the worker alive.
+            for request in batch:
+                if not request.future.done():
+                    request.future.set_exception(exc)
+            return
+        for request, result in zip(batch, results):
```

Ruff (0.15.17)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)